### DISCUSSION TITLE
Vocaster - Enhance feature

### DISCUSSION THREAD

**Discussion by ufromy** [geoffreybennett/alsa-scarlett-gui#153] [Ideas]
![Copie d'écran_20241217_145828](https://github.com/user-attachments/assets/667d502b-ae13-435a-8e37-eb519a6b933a)

Unless I am mistaken you can not select between the different enhance feature in the Vocaster one and two series. Is there anything I am missing or can this be added ? Great work between ! Love your work.

**Comment by geoffreybennett** [comment]
Hi @ufromy,

The GUI does not yet support changing the DSP parameters unfortunately, but the driver does have support for it. The UI for tweaking the presets is a strong contender for World's-Worst-UI, but I have created a shell script that will let you reasonably easily switch between the radio/clean/warm/bright presets. Please have a read through https://github.com/geoffreybennett/alsa-scarlett-gui/issues/58. The script is for the Vocaster One; updating it for the Vocaster Two shoudn't be too hard, just apply the settings to Line 2 as well as Line 1. Let me know if you need assistance with that.

Regards,
Geoffrey.

**Comment by wchouser3** [comment]
Is there a plan to add this functionality to the UI? I'm going to go try your script now. I have the Vocaster One
